### PR TITLE
chore: add mmarchini as calendar maintainer

### PR DESCRIPTION
**Pull request by mmarchini**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ This list should be reviewed and pruned annually (at minimum). The calendar has
 - [@MylesBorins](https://github.com/MylesBorins) - **Myles Borins**
 - [@ryanmurakami](https://github.com/ryanmurakami) - **Ryan Lewis**
 - [@trott](https://github.com/trott) - **Rich Trott**
+- [@mmarchini](https://github.com/mmarchini) - **Mary Marchini**
```
Comment (richardlau) [inline README.md:90]: Comment above this block says this should be sorted by GitHub handle.

**Comment by nschonni** [comment]
As a later thing, does it make sense to have a GitHub team and use the ncu-sync?

**Review by richardlau — APPROVED** [review]

**Review by Trott — APPROVED** [review]

**Review by cjihrig — APPROVED** [review]

**Review by gireeshpunathil — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mhdawson** [comment]
@nschonni I'm not sure the github team helps in this case as the team would not actually be used for anything in the repo and a separate manual step is still required to give the person calendar access.

**Comment by mhdawson** [comment]
@mmarchini added you as a calendar maintainer so you should have access now.

**Pull request merged**